src/components/Footer.tsx

```diff
@@ -0,0 +1,52 @@
+import { FilterType } from '../types/FilterType';
+import { Todo } from '../types/Todo';
+import React, { SetStateAction, Dispatch } from 'react';
+
+type FooterProps = {
+  todos: Todo[];
+  filterType: FilterType;
+  setFilter: Dispatch<SetStateAction<FilterType>>;
+};
+
+const filtersButtons = ['All', 'Active', 'Completed'];
+
+export const Footer: React.FC<FooterProps> = ({
+  todos,
+  filterType,
+  setFilter,
+}) => {
+  const activeTodos = todos.filter(t => !t.completed);
```

Comment:
One-letter arguments in function are not a good idea. Please make it more readable.